### DISCUSSION TITLE
Multi-Window App with Regions on Multi-ContentPages

### DISCUSSION THREAD

**Discussion by pAtMaN-bcassa** [PrismLibrary/Prism#3241] [.NET MAUI] [answered]
I am currently developing a project for Windows and MacOS with Prism.Maui including regions, and I need a second / third window that should be displayed on another monitor and also contain regions for navigation.

Using the IWindowManager interface, I can, of course, open a newly created window, and the bindings work as well. However, it seems that the window is not properly registered because it does not include the regions of the SecondPage in the RegionManager. RegisterForNavigation for MainPageView and SecondPageView has also been done, but the thing with IWindowManager is quite a different navigation system compared to the NavigationService or the RegionManager. 

Should I use the DialogService for this? Is it possible, like in Prism.WPF, to register the DialogWindow or transfer the dialog to the second monitor using X, Y coordinates?

Has anyone successfully implemented this before?

**Comment by dansiegel** [comment, marked as answer]
The IWindowManager is meant for internal use and generally isn't something you should be trying to use. That said Prism 9.0 does NOT support multi-window. It is something that is planned for 9.1

**Reply by pAtMaN-bcassa** [reply]
Thank you for the quick response.
